chapters/ko/_toctree.yml

```diff
@@ -26,4 +26,23 @@
   - local: chapter1/10
     title: 단원 마무리 퀴즈
     quiz: 1
-    
\ No newline at end of file
+
+- title: 2. 🤗 Transformers 사용하기
+  sections:
+  - local: chapter2/1
+    title: 단원 소개
+  # - local: chapter2/2
```

Comment:
Could we please remove the commented out sections to avoid merge conflicts with other contributors?

Reply:
Nice to meet you, @lewtun! I removed the comments. Happy to contribute :)